### ISSUE TITLE
the skeleton created by bailador does not work in 0.0.7

### ISSUE THREAD

**Issue by szabgab** [Bailador/Bailador#189]
```
PERL6LIB=~/work/Bailador/lib perl6 Bailador/bin/bailador --name=Bailador-Chat new
cd Bailador-Chat
PERL6LIB=~/work/Bailador/lib perl6 ~/work/Bailador/bin/bailador watch bin/app.pl6
```

The error I get:
```
Failed to open file /Users/gabor/work/Bailador-Chat/bin/views/index.tt
```
So it thinks the views directory is inside the bin directory.

**Comment by indhare** [comment]
work around:

# cd my-app/bin
# ln -s ../views
# bailador watch

**Comment by szabgab** [comment]
Fixed and tested in t/20-cli.t but see also related #169

**Comment by szabgab** [comment]
The solution I provided seems to be problematic in other cases. Let's review this and let's see that #169 which might solve the problem.

**Comment by ufobat** [comment]
how was it problematic in other cases?

**Comment by szabgab** [comment]
When I left out the `Bailador::import();` from the `bin/app.pl` file, it mapped the `views` incorrectly.

**Comment by ufobat** [comment]
closed because #169 fixed it.
